src/udev/udevd.c

```diff
@@ -1448,10 +1448,11 @@ static int on_sigchld(sd_event_source *s, const struct signalfd_siginfo *si, voi
                         device_tag_index(worker->event->dev, NULL, false);
 
                         if (manager->monitor) {
-                                /* forward kernel event without amending it */
+                                /* Forward kernel event unchanged */
```

Comment:
btw, I think we should revisit this bit, i.e. when we forward the thing after the worker failed, I think we should say so in the event. i.e. add a property "UDEV_WORKER_FAILED=1" or so (and maybe even the exit status/code as separate props).

That's useful for debugging, but also is less confusing for apps: if apps see these events they know why their props are missing.

Reply:
Interesting. But let's do that in another later PR.